.github/workflows/manual-publish.yml

```diff
@@ -36,12 +46,14 @@ jobs:
         id: publish
         with:
           dry-run: ${{ inputs.dry-run }}
+          dry-run-gh: ${{ inputs.dry-run-gh }}
           token: ${{ secrets.GITHUB_TOKEN }}
           homebrew-gh-secret: ${{secrets.HOMEBREW_DEPLOY_KEY}}
           tag: ${{ inputs.tag }}
 
   release-ldcli-npm:
     runs-on: ubuntu-latest
+    if: ${{ inputs.dry-run-npm == false }}
```

Comment:
This should work correctly since `input` values in workflows - if a boolean type - should actually be a boolean.